i18n/zh/docusaurus-plugin-content-docs/current/installation/installation.md

```diff
@@ -6,12 +6,12 @@ title: 安装概述
 
 ### Karmada kubectl 插件
 `kubectl-karmada` 是允许你控制 Karmada 控制面的 Karmada 命令行工具，该工具表现为一个 [kubectl 插件][1]。
-有关安装说明，请参见[安装 kubectl-karmada](./install-cli-tools.md#install-kubectl-karmada)。
+有关安装说明，请参见[安装 kubectl-karmada](./install-cli-tools.md#安装-kubectl-karmada)。
```

Comment:
BTW, cc @windsonsea for a note, when we translate a document we may need to pay attention to the relative link which should also be translated.